7 hours/blocking-nonblocking/blocking.js

```diff
@@ -0,0 +1,6 @@
+var fs = require('fs');
+var data = fs.readFileSync('data.txt')
```

Comment:
// this is the blocking line
```suggestion
var data = fs.readFileSync('data.txt')
```